pkg/services/accesscontrol/pluginutils/utils_test.go

```diff
@@ -0,0 +1,78 @@
+package pluginutils
+
+import (
+	"testing"
+
+	"github.com/grafana/grafana/pkg/plugins"
+	ac "github.com/grafana/grafana/pkg/services/accesscontrol"
+	"github.com/stretchr/testify/require"
+)
+
+func TestToRegistrations(t *testing.T) {
+	tests := []struct {
+		name string
+		regs []plugins.RoleRegistration
+		want []ac.RoleRegistration
+	}{
+		{
+			name: "no registration",
+			regs: nil,
+			want: []ac.RoleRegistration{},
+		},
+		{
+			name: "no registration",
```

Comment:
nit: the description should probably be "registration gets converted successfully" or something like that.